src/Microsoft.IdentityModel.Tokens/Base64UrlEncoding.cs

```diff
@@ -337,15 +222,12 @@ public static string Encode(byte[] input, int offset, int length)
         /// <param name="strSpan">The string represented by a span to validate.</param>
         /// <param name="offset">The index of the character in <paramref name="strSpan"/> to start the decode operation.</param>
         /// <param name="length">The number of characters in <paramref name="strSpan"/> to decode, starting from <paramref name="offset"/>.</param>
-        /// <returns>The size of the decoded bytes array.</returns>
-        internal static int ValidateAndGetOutputSize(ReadOnlySpan<char> strSpan, int offset, int length)
```

Comment:
This is used in IM.JsonWebTokens. So needs to be reverted otherwise we'll run into the package version mismatch error. You can probably just keep this as-is and add a comment that it's obsolete and the new Validate method should be used.